robotino_calibration_sim/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotino_calibration_sim)
```

Comment:
please add the dependency robotino_calibration_sim to the metapackage  https://github.com/squirrel-project/squirrel_calibration/blob/indigo_dev/squirrel_calibration/package.xml#L15